engine/settings/base.py

```diff
@@ -541,6 +541,7 @@ class BrokerTypes:
 
 FEATURE_CAPTCHA_ENABLED = getenv_boolean("FEATURE_CAPTCHA_ENABLED", default=False)
 RECAPTCHA_SECRET_KEY = os.environ.get("RECAPTCHA_SECRET_KEY")
```

Comment:
ohh interesting. I think these two secrets, `FEATURE_CAPTCHA_ENABLED` and `RECAPTCHA_SECRET_KEY` are not used (AFAICT). I wonder consider getting rid of these and moving `RECAPTCHA_SITE_KEY` down with the `DRF_RECAPTCHA_*` variables further down in the file